client/testutil/driver_compatible.go

```diff
@@ -8,15 +8,20 @@ import (
 )
 
 func ExecCompatible(t *testing.T) {
-	if runtime.GOOS != "windows" && syscall.Geteuid() != 0 {
+	if runtime.GOOS == "windows" || syscall.Geteuid() != 0 {
```

Comment:
Can you reset this back to what it was. This always skips the tests on windows. It should only be skipped if not root when not on windows.